eyelid-match-ops/src/primitives/yashe/conf.rs

```diff
@@ -153,22 +276,40 @@ where
 ///
 /// This uses the full number of iris bits, which gives an upper bound on benchmarks.
 impl YasheConf for IrisBits {
+    type PolyBN = IrisBitsBN;
+
     const T: u64 = 2048;
 }
 
 /// Full resolution polynomial parameters.
 ///
 /// These are the parameters for full resolution, according to the Inversed Tech report.
 impl YasheConf for FullRes {
+    type PolyBN = FullResBN;
+
+    // VERIFY: max T should be 2^15, not 2^10
     const T: u64 = 1024;
 }
 
+/// Middle resolution polynomial parameters.
+///
+/// These are the parameters for middle resolution, according to the Inversed Tech report.
+impl YasheConf for MiddleRes {
+    type PolyBN = MiddleResBN;
+
+    // VERIFY: max T should be 2^12, not 2^8
+    const T: u64 = 256;
+}
```

Comment:
What needs to be done to finish this, @emmorais?

Reply:
I think an issue will be the best way to proceed.

Reply:
This is now ticket #100